src/pages/[platform]/how-amplify-works/faq/index.mdx

```diff
@@ -29,31 +29,43 @@ export function getStaticProps(context) {
 
 We are working on supporting migrations from Gen 1 to Gen 2. In the meantime, Gen 1 is still fully supported and you can continue to use it in apps you've already created.
 
+**If I have a Gen 1 app, can I use Gen 2 in it?**
+
+Amplify Gen 1 and Gen 2 follow different architectural and tooling paradigms, which was necessary to address common customer feedback from Gen 1. You will need to use our upcoming migration tooling to move from a Gen 1 to Gen 2 app. You cannot use Amplify Gen 1 (Studio/CLI) in the same app as Gen 2.
+
 **Should I use Amplify Gen 1 or Gen 2 in new apps?**
 
 If you're building a new app, we recommend you use Amplify Gen 2.
 
 **Does Amplify Gen 2 support DataStore?**
 
-Amplify code-first DX (Gen 2) supports GraphQL APIs without DataStore. We will release migration support for moving DataStore Gen 1 apps to Gen 2.
+Amplify Gen 2 supports GraphQL APIs without DataStore. We will release migration support for moving DataStore Gen 1 apps to Gen 2.
 
 **What programming languages does Amplify Gen 2 support?**
 
 Amplify Gen 2 supports a wide range of programming languages for client-side development. This includes dedicated client-side libraries for JavaScript, TypeScript, Dart, Java, Kotlin, and Swift. For backend development, Amplify Gen 2 uses TypeScript.
 
 **In Gen 1, Amplify offered a set of use case categories for building applications (for example, Authentication, Analytics, API, DataStore, Geo, and Predictions). Are those same categories available in Gen 2?**
 
-Amplify code-first DX (Gen 2) supports Auth, Data, Storage, and Functions. Other use cases can be implemented using AWS Cloud Development Kit (AWS CDK) constructs which there is documentation for under the respective category name.
+Amplify Gen 2 offers built-in support for Auth, Data, Storage, and Functions. Other use cases can be implemented in Amplify Gen 2 as well using AWS Cloud Development Kit (AWS CDK) constructs which there is documentation for under the respective category name.
 
 **What if we want to add a feature like AI/ML or Amazon Location Service to our application in Gen 2?**
 
 Because Amplify builds on the AWS CDK, any of the 175+ AWS services can be added to your app using [custom resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) and L2/L1 AWS CDK constructs.
+
+**What happens once my application grows too big and I want to do more configuration with my application (add more features, other AWS services, etc.)?*
+
+Amplify is built on top of the AWS CDK, so you can add any [AWS service to your Amplify app](/[platform]/build-a-backend/add-aws-services/custom-resources/). You can also override [Amplify-generated configuration of your resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) using the CDK. We also offer standalone L3 CDK constructs for Amplify's Auth, Data, and Storage use cases that you can use within any existing CDK application.
```

Comment:
Think you need to assert the answer here is "You can stay with Amplify no matter how big your application grows". Points to be made (please improve language): 1/ Amplify is layered on CDK and CloudFormation, providing standard convention for working with AWS services 2/ You can use any deployment pipeline of your choice if you want more control over your CI.

Reply:
Updated